tests/geo-index.test.js

```diff
@@ -35,10 +35,10 @@ describe('geoindex', function () {
         assert.deepEqual(generatedIndex, expectedIndexData)
 
         // also make sure certain subzone is written
-        fs.stat(TEST_DATA_DIR + '/b/b/d/c/d/d/geo.buf',
+        fs.stat(TEST_DATA_DIR + '/geo.dat',
           function (err, stats) {
             assert.isNotOk(err)
-            var data = new Pbf(fs.readFileSync(TEST_DATA_DIR + '/b/b/d/c/d/d/geo.buf'))
+            var data = new Pbf(fs.readFileSync(TEST_DATA_DIR + '/geo.dat'))
```

Comment:
I think this is a good effort to keep this test case updated and running. However, I think it happens to be a matter of luck that this passes since the geobuf.decode of the .dat file is probably luckily decoding the correct feature at the front of the dat file.

A better test case would extract the exact relevant length of data from the .dat file, and also do it for two different sub-areas to be sure.

I have gone ahead and pushed a branch ([improve-geoindex-test-case](https://github.com/evansiroky/node-geo-tz/tree/improve-geoindex-test-case)) with an example of how this test case can include asserting two different sub-areas. It would be great if this PR could bring in that refactored test case and then adapt as suggested.

Reply:
Ah, yeah, I thought that was kind of weird, too, but I forgot to dig deeper. Fixed.